### PR TITLE
Add two channel and manage them for push notif

### PR DESCRIPTION
**Pull request by HugoGresse**
- add two new channel id (and change the android default one) in order to have one channel with sound and very important, the other less important without sound. 
- change notification push type if the room is starting in less than 5 min. If it's more, it a reminder push notif, otherweise it's a sound notif with "URGENCY". 
- manage channel id in functions

### CODE REVIEW

File: src/services/local-notifications.ts [hunk 1]
```diff
@@ -5,7 +5,12 @@ import {
 } from '@capacitor/core'
 const { LocalNotifications } = Plugins
 export class LocalNotificationsService {
-    static async schedule(title?: string, body?: string, extra?: any) {
+    static async schedule(
+        title?: string,
+        body?: string,
+        extra?: any,
+        channelId: string = 'visio-call-notifications'
```
Comment (MattiaPrimavera) [inline src/services/local-notifications.ts:12]: By default should not simulate a call in my advice

File: src/services/push-notifications.ts [hunk 2]
```diff
@@ -13,18 +13,30 @@ const { PushNotifications } = Plugins
 export class PushNotificationsService {
     static createDefaultChannel() {
         const notificationChannel: NotificationChannel = {
-            id: 'visio-call-notifications', // id must match android/app/src/main/res/values/strings.xml's default_notification_channel_id
-            name: 'Visio call notifications',
-            description: 'Visio call notifications',
+            id: 'visio-call-notifications',
```
Comment (MattiaPrimavera) [inline src/services/push-notifications.ts:16]: Same as above

**Review by MattiaPrimavera — CHANGES_REQUESTED** [review]

File: src/services/push-notifications.ts (hunk 2 above)
Comment (HugoGresse) [reply src/services/push-notifications.ts:16]: I've changed the default_channel_id for the android manifest which remove any specifics of the default channel and is not used in the app

File: src/services/local-notifications.ts [hunk 3]
```diff
@@ -18,7 +23,7 @@ export class LocalNotificationsService {
                     actionTypeId: '',
                     extra,
                     smallIcon: 'ic_logo_mobile',
-                    channelId: 'visio-call-notifications',
+                    channelId: channelId,
```
Comment (ntchou) [inline src/services/local-notifications.ts:26]:
```suggestion
                    channelId,
```

File: src/services/local-notifications.ts (hunk 1 above)
Comment (ntchou) [inline src/services/local-notifications.ts:12]: Type is inferred. No need to precise

**Review by ntchou — APPROVED** [review]

**Pull request merged**